ui/client/actions/actions.js

```diff
@@ -464,9 +461,14 @@ export function displayProcessCounts(processCounts) {
 }
 
 export function urlChange(location) {
-  return {
-    type: 'URL_CHANGED',
-    location: location
+
+  return (dispatch) => {
+    dispatch(handleHTTPError(null))
```

Comment:
It's for clearing error status, right?

Reply:
Exacly! When you change url then error should disappear.